lib/engine/game/base.rb

```diff
@@ -1105,7 +1105,7 @@ def all_companies_with_ability(ability_type)
       end
 
       def payout_companies
-        companies = @companies.select { |c| c.owner && c.revenue.positive? }
+        companies = @companies.select { |c| c.owner && c.owner != bank && c.revenue.positive? }
```

Comment:
Override rather than alter.

Reply:
This is just to prevent spurious message. It's not specific for this game at all

Reply:
then this should be a separate PR then